compute/admin_guide/agentless-scanning/agentless-scanning.adoc

```diff
@@ -1,11 +1,11 @@
 == Agentless Scanning
 
-Agentless scanning lets you inspect the risks and vulnerabilities of a virtual machine without having to install an agent or affecting the execution of the instance.
+Agentless scanning lets you inspect the risks and vulnerabilities of a cloud workloads without having to install an agent or affecting the execution of your workload.
 Prisma Cloud gives you the flexibility to choose between agentless and agent-based security using Defenders.
-Currently, Prisma Cloud supports agentless scanning on AWS, GCP and Azure hosts for vulnerabilities and compliance.
-Agentless capabilities will continue to be enhanced over future releases in parallel with Defender capabilities.
+Prisma Cloud supports agentless scanning cloud workloads on AWS, Azure, GCP, and OCI for vulnerabilities and compliance.
+In AWS, Azure, and GCP you can also use agentless scanning for container images.
 
-Continue reading this page to learn about the Agentless Scanning architecture and results. 
+Continue reading this page to learn about the agentless scanning modes, architecture, and results.
 To configure Agentless Scanning for your accounts, follow the xref:./onboard-accounts/onboard-accounts.adoc[step-by-step instructions].
```

Comment:
We can remove this. The links are there in the page automatically because panconv adds it, and I see it in the PDF.